Automattic/wp-calypso
# Nav Redesign: Move search and notifications to footer

Fixes https://github.com/Automattic/dotcom-forge/issues/6966

## Proposed Changes

Moves the search and notifications buttons to the footer of the new sidebar so the Sites, Domains, and Reader links are kept in the same position when the sidebar is expanded/collapsed.

The sizing/spacing of the footer has been slightly adjusted as well to ensure that all items are correctly aligned.

**Before**

Expanded | Collapsed
--- | ---
<img width="147" alt="Screenshot 2024-05-08 at 12 16 42" src="https://github.com/Automattic/wp-calypso/assets/1233880/7cf9f08f-1e1f-4460-98ce-db56d7132161"> | <img width="35" alt="Screenshot 2024-05-08 at 12 16 57" src="https://github.com/Automattic/wp-calypso/assets/1233880/be8c99f5-ff2e-4f36-a01e-ddfd005d71ad">

**After**

Expanded | Collapsed
--- | ---
<img width="147" alt="Screenshot 2024-05-08 at 12 17 19" src="https://github.com/Automattic/wp-calypso/assets/1233880/86106802-ba5c-4ef3-bb29-73cd4bdfa9c2"> | <img width="35" alt="Screenshot 2024-05-08 at 12 17 26" src="https://github.com/Automattic/wp-calypso/assets/1233880/6a2fb6b9-a34c-4b29-af2e-2f912716f1b8">


I also took the opportunity to refine the position of the tooltips so they look more polished.

**Before**

https://github.com/Automattic/wp-calypso/assets/1233880/4da315c1-f385-4be7-8058-d50a3abe0248

**After**

https://github.com/Automattic/wp-calypso/assets/1233880/99cb99e9-1d3f-4ad4-a06a-9484cb2f1769





## Testing Instructions

- Use the Calypso live link below
- Go to `/sites`
- Make sure the search and notifications icons are now placed at the bottom
- Select a site to collapse the sidebar
- Make sure the footer looks good
- Make sure tooltips also look good
